### PR TITLE
Docs: add a notice in helm tutorial

### PR DESCRIPTION
**Pull request by LinzeSu**
Signed-off-by: Lingzhe Sun <lingzhesun@gmail.com>

### Description of your changes
Add a notice in the Before Starting section. As the current app-with-chart-redis.yaml would fail because the chart version is outdated.

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Update `sidebar.js` if adding a new page.
- [ ] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

File: docs/tutorials/helm.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Starting from here, you will learn to use the KubeVela Addons to install plug-in
 ## Before starting
 
 - Choose a Helm Chart you want to deploy. In the tutorial, we take [bitnami/redis](https://github.com/bitnami/charts/tree/master/bitnami/redis) as an example.
+- If you are using [bitnami/redis](https://github.com/bitnami/charts/tree/master/bitnami/redis), check if the chart version you specified is still listed [here](https://artifacthub.io/packages/helm/bitnami/redis) and available.
```
Comment (wonderflow) [inline docs/tutorials/helm.md:14]: please also help updateing the redis version below. thanks

Comment (wonderflow) [reply docs/tutorials/helm.md:14]: please also update the zh verison, thanks

**Review by wonderflow — APPROVED** [review]

**Comment by chivalryq** [comment]
@LinzeSu Please sign the DCO.

**Comment by LinzeSu** [comment]
> @LinzeSu Please sign the DCO.

Hi, sorry for late reply. Actually I did sign. But the fail message indicates that github encrypted my email and failed to verify. Can you merge manually, or else I need to perform a rebase operation.

**Comment by chivalryq** [comment]
@LinzeSu Sorry I can't merge it. It's a required check. You can follow the instruction in the https://github.com/kubevela/kubevela.io/pull/1161/checks. Rebase commits and squash it to one.

**Pull request closed without merging**